content/doc/book/installing/windows.adoc

```diff
@@ -56,7 +56,7 @@ To run Jenkins service using a *local or domain user*, specify the domain user n
 password with which you want to run Jenkins,
 click on *Test Credentials* to test your domain credentials and click on *Next*.
 
-image:tutorials/windows-jenkins-logon-credentials.jpg[alt="Jenkins Service Logon Credentials",width=80%]
+image:tutorials/windows-jenkins-logon-credentials.png[alt="Jenkins Service Logon Credentials",width=80%]
```

Comment:
I added this change to the pull request so that the new screenshot would be displayed rather than continuing to see the old screenshot.  You can see the results of that change at https://deploy-preview-6608--jenkins-io-site-pr.netlify.app/doc/book/installing/windows/ after the preview site has been updated by the Jenkins job.,